core/src/main/scala/org/apache/spark/io/CompressionCodec.scala

```diff
@@ -113,7 +118,6 @@ private[spark] object CompressionCodec {
     }
   }
 
-  val FALLBACK_COMPRESSION_CODEC = "snappy"
```

Comment:
Please recover this variable, `FALLBACK_COMPRESSION_CODEC`, @beliefer .

Reply:
Uh. `FALLBACK_COMPRESSION_CODEC` is a public constant.